### DISCUSSION TITLE
Request Camera permission

### DISCUSSION THREAD

**Discussion by islamzaoui** [tauri-apps/tauri#7615] [Q&A]
Hello guys
I have a fully working Tauri app but I have a problem with permission to use the camera if the user clicks the block button he will stick unable to use the camera until he deletes the EBWebView folder but this is not user-friendly and it will delete indexedDB. 
WebView2 have permission APIs, how i can use it with Tauri?

note: i use tauri with sveltekit

**Comment by krokosik** [comment]
Check out this issue https://github.com/tauri-apps/tauri/issues/4434
